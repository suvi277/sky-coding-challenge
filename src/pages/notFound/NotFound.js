import React from 'react';
import MainLayout from 'components/hoc/Layout';
import PageHeading from 'components/pageHeading/PageHeading';

const NotFound = () => {
	return (
		<MainLayout>
			<PageHeading title="Oops Page Not Found" />
		</MainLayout>
	);
};
export default NotFound;
