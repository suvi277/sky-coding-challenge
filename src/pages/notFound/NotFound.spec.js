import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound Component', () => {
	const wrapper = shallow(<NotFound />);

	it('renders as expected', () => {
		expect(wrapper).toMatchSnapshot();
	});

	it('should exist MainLayout Component', () => {
		expect(wrapper.find('MainLayout')).toHaveLength(1);
	});

	it('should exist PageHeading Component', () => {
		expect(wrapper.find('PageHeading')).toHaveLength(1);
	});
});
