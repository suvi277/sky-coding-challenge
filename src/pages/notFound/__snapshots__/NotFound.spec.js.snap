// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component renders as expected 1`] = `
<MainLayout>
  <PageHeading
    title="Oops Page Not Found"
  />
</MainLayout>
`;
